content/reference/endpoints/rates_tables.md

```diff
@@ -28,6 +29,7 @@ Name             | Type    | Read/Write | Description
 id               | Integer | Read       | Rates Table's id.
 -----------------|---------|-------------------------
 name             | String  | Read/Write | **Required**. Rates Table's name. (50 characters max)
+public_notes     | Text    | Read       | Rates Table's public notes.
```

Comment:
I think we should use JSON types only, right? So `String` rather than `Text`

Reply:
@StoneFrog , I see `Type = Text` on other entities as well. Are you sure we should use `String` ?

Reply:
I'd keep text, it's obvious that text is longer than string, but we can add a section about what our types mean in the doc if you think this needs clarification.

Reply:
@Azdaroth , WDYT?

Reply:
"String" would be more correct, but I think "Text" is more practical, same as Integer vs. Biginteger.

Reply:
I'm ok with keeping text but good to explain it. We already have formats section explaining time decimal and so on.